src/main/java/org/mule/extension/sftp/internal/connection/SftpClient.java

```diff
@@ -80,7 +80,7 @@ public class SftpClient {
   private static final Logger LOGGER = getLogger(SftpClient.class);
   protected static final OpenMode[] CREATE_MODES = {OpenMode.Write, OpenMode.Create};
   protected static final OpenMode[] APPEND_MODES = {OpenMode.Write, OpenMode.Append};
-  public static final Long PWD_COMMAND_EXECUTION_TIMEOUT = 30L;
+  private static final Long PWD_COMMAND_EXECUTION_TIMEOUT = 30L;
```

Comment:
Where is this constant used? Is it accessed from somewhere outside the SftpClient class?

Reply:
No, I see that is only in SftpClient class, used by method executePWDCommandWithTimeout()